opennms-install/src/main/java/org/opennms/install/Installer.java

```diff
@@ -414,6 +415,19 @@ public void loadProperties() throws IOException {
         //        m_installerDb.setCreateSqlLocation(m_etc_dir + File.separator + "create.sql");
     }
 
+    private void loadScvProperties() throws IOException{
+
+        loadEtcPropertiesFile(Path.of("opennms.properties.d","scv.properties").toString());
+
```

Comment:
Check opennms.properties and all files in opens.properties.d, see Bootstrap getPropertiesFiles() method as an example.